### PR TITLE
Enable native library loading via Java path

### PR DESCRIPTION
**Pull request by samuel-rufi**

### CODE REVIEW

**Review by Thoralf-M — DISMISSED** [review]

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Add a changelog entry please

File: bindings/java/iota-wallet-java/CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 1.0.0-rc.2 - 2022-11-15
```
Comment (thibault-martinez) [inline bindings/java/iota-wallet-java/CHANGELOG.md]: rc3 + no date

File: bindings/java/iota-wallet-java/CHANGELOG.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
+### Added
+
+- Enable native library loading via Java path;
```
Comment (thibault-martinez) [inline bindings/java/iota-wallet-java/CHANGELOG.md:26]: Is the `NativeApi.ClientCommand` to `WalletCommand` related to that or that's something else ?

File: bindings/java/iota-wallet-java/CHANGELOG.md (hunk 1 above)
Comment (samuel-rufi) [reply bindings/java/iota-wallet-java/CHANGELOG.md]: fixed

File: bindings/java/iota-wallet-java/CHANGELOG.md (hunk 2 above)
Comment (samuel-rufi) [reply bindings/java/iota-wallet-java/CHANGELOG.md:26]: Its related, I clarified it in the new changelog

**Review by thibault-martinez — APPROVED** [review]

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**